menu/menu_miiverse_confirm.cpp

```diff
@@ -52,7 +52,7 @@ void RenderMenuMiiverseConfirm(const BackupState& state) {
 
     OSScreenPutFontEx(SCREEN_TV, 0, 26, "HOME: Cancel");
     OSScreenPutFontEx(SCREEN_TV, 80, 26, "A: Apply Patches");
-
+    OSScreenPutFontEx(SCREEN_TV, 60, 26, "Y: Restore Patches");
```

Comment:
"Uninstall patches" or "Remove patches" would probably be better, "Restore" sounds like installing the patches again.

Reply:
Ok